### PR TITLE
[Java, doc] add new calls for crash reporting

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Review by ijunaid — APPROVED** [review]

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
this doesn't fully explain the fatal part of a handled exception and the point of the recordUnhandled call

File: java/next.md [hunk 1]
```diff
@@ -129,19 +129,41 @@ Config config = new Config("http://YOUR.SERVER.COM", "YOUR_APP_KEY", targetFolde
...
+</p>
+<pre>config.disableUnhandledCrashReporting();</pre>
 <h2 id="h_01HD1AK4K4M40M1J7W0R50QGQM">Handled Exceptions</h2>
 <p>
   You may catch an exception during application runtime. You might report with
   these functions:
 </p>
-<pre><code class="java">Countly.instance().addCrashReport(Throwable t, boolean fatal);</code></pre>
+<pre><code class="java">Countly.instance().crashes().recordHandledException(Throwable t);
+// you can also add segments values to inspect it detailed way later on
```
Comment (turtledreams) [inline java/next.md:146]: "inspect it detailed way later on" should go I think

File: java/next.md [hunk 2]
```diff
@@ -129,19 +129,41 @@ Config config = new Config("http://YOUR.SERVER.COM", "YOUR_APP_KEY", targetFolde
...
+<pre><code class="java">Countly.instance().crashes().recordUnhandledException(Throwable t);
+// you can also add segments values to inspect it detailed way later on
+Countly.instance().crashes().recordUnhandledException(Throwable t, Map&lt;String, Object&gt; segment);</code></pre>
+<h2 id="h_01HG0S5QWDC5WEQSV0W724XCG4">Crash Breadcrumbs</h2>
+<p>
+  Throughout your app you can leave crash breadcrumbs which would describe previous
+  steps that were taken in your app before the crash. After a crash happens, they
+  will be sent together with the crash report.
+</p>
+<p>Following the command adds crash breadcrumb:</p>
```
Comment (turtledreams) [inline java/next.md:161]: following?

File: java/next.md (hunk 1 above)
Comment (turtledreams) [reply java/next.md:146]: oh it was a copy pasta issue from "inspect it in a detailed way later on,"

File: java/next.md [hunk 3]
```diff
@@ -129,19 +129,41 @@ Config config = new Config("http://YOUR.SERVER.COM", "YOUR_APP_KEY", targetFolde
...
   You may catch an exception during application runtime. You might report with
   these functions:
 </p>
-<pre><code class="java">Countly.instance().addCrashReport(Throwable t, boolean fatal);</code></pre>
+<pre><code class="java">Countly.instance().crashes().recordHandledException(Throwable t);
+// you can also add segments values to inspect it detailed way later on
+Countly.instance().crashes().recordHandledException(Throwable t, Map&lt;String, Object&gt; segment);</code></pre>
+<p>
+  If an exception handled and it turns out be a fatal exception for the application,
+  it can be reported with these functions:
```
Comment (turtledreams) [inline java/next.md:150]: rework this sentence

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**